.github/workflows/release_check.yml

```diff
@@ -0,0 +1,29 @@
+name: Run JS SDK test
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '**/setup.py'
+  pull_request:
+    paths:
+      - '**/setup.py'
+  workflow_dispatch:
+
+jobs:
+  generate-types:
+    defaults:
+      run:
+        working-directory: ./python
+    - name: Run `composio apps generate-types`
+      run: |
+          python -m pip install uv
+          python -m uv pip install .
+          python -m uv pip install black
+          python -m uv run composio apps generate-types
+          # Run black over the files before checking if they changed
+          python -m uv run black composio/client/enums
+
+    - name: Check if type stubs changed, and fail if they did
+      run: git diff --exit-code
```

Comment:
Consider adding an informative error message when the type stubs have changed. Something like:
```yaml
run: |
  if ! git diff --exit-code; then
    echo "::error::Type stubs are not up to date. Please run 'composio apps generate-types' locally and commit the changes."
    exit 1
  fi
```